dataproc_jupyter_plugin/services/cluster.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2023 Google LLC
```

Comment:
We already have a file in the `services` directory for Dataproc called `dataproc`.

Move all of the methods from this file into that one and then delete this entire file.